cat.txt

```diff
@@ -0,0 +1,28 @@
+       ,
```

Comment:
I like how you used txt files because mine being printed from the main file was annoying and then you can use more complex art